assets/js/atomic/blocks/product-elements/image/supports.ts

```diff
@@ -19,7 +20,7 @@ export const supports = {
 			fontSize: true,
 			__experimentalSkipSerialization: true,
 		},
-		...( hasSpacingStyleSupport() && {
+		...( __experimentalGetSpacingClassesAndStyles === 'function' && {
```

Comment:
The `tyepeof` keyword is missing:

```suggestion
		...( typeof __experimentalGetSpacingClassesAndStyles === 'function' && {
```